### ISSUE TITLE
Long @field type not supported

### ISSUE THREAD

**Issue by mayankasthana** [fsanaulla/chronicler#202]
The macros documentation lists `Long` in the supported types -  https://github.com/fsanaulla/chronicler/blame/master/docs/macros.md#L32.

However, any case class with `@field` Long type, fails compilation.

https://github.com/fsanaulla/chronicler/blob/af1bc06e635650ac9c73b2ce0e48b06eb3fbb2fb/modules/macros/src/main/scala/com/github/fsanaulla/chronicler/macros/InfluxImpl.scala#L51-L52
does not list `getType[Long]`.

**Comment by fsanaulla** [comment]
Hey @mayankasthana. Let me check it. I'll write you back in a while

**Comment by fsanaulla** [comment]
@mayankasthana Can you please provide a short example of it?

**Comment by mayankasthana** [comment]
```
import com.github.fsanaulla.chronicler.core.enums.Consistencies
import com.github.fsanaulla.chronicler.macros.annotations.{field, tag}
import com.github.fsanaulla.chronicler.urlhttp.io.InfluxIO
import com.github.fsanaulla.chronicler.urlhttp.shared.InfluxConfig
import com.github.fsanaulla.chronicler.macros.auto._


object Main extends App {
  case class ProgressEvent(@tag name: String,
                           @field inputRecords: Int,
                           @field outputRecords: Long)

  val influxConfig = InfluxConfig("localhost", 8086)
  val influx = InfluxIO(influxConfig)
  val measurement: influx.Measurement[ProgressEvent] =
    influx.measurement("db", "measurement")
  println(measurement.write(ProgressEvent("name1", 100, 4000L), Consistencies.One).isSuccess)
}
```

Compilation error-
```
Error:(19, 28) Unsupported type for @field outputRecords: Long
  println(measurement.write(ProgressEvent("name1", 100, 4000L), Consistencies.One).isSuccess)
```

**Comment by mayankasthana** [comment]
The error should be coming from
https://github.com/fsanaulla/chronicler/blob/af1bc06e635650ac9c73b2ce0e48b06eb3fbb2fb/modules/macros/src/main/scala/com/github/fsanaulla/chronicler/macros/InfluxImpl.scala#L369

**Comment by fsanaulla** [comment]
Should be solved in #203

**Comment by fsanaulla** [comment]
@mayankasthana it should be fixed in 0.6.3. Also you can try it now.

**Comment by mayankasthana** [comment]
Tested, works!

Thanks a lot for the quick fix.
